lib/config.py

```diff
@@ -100,6 +100,25 @@
     ('--packages-dir', '-d'):
         dict(help='Directory of packages used in the ISO image.',
              default='./result'),
+    ('--iso-name',):
+        dict(help='ISO name.',
+             default='OpenPOWER-Host_OS'),
```

Comment:
There's no need to have those defaults here. They are already in the config.yaml file.